app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
         <% end %>
         <span class="caret"></span>
       </a>
-      <ul class="dropdown-menu" role="menu">
+      <ul class="dropdown-menu finders" role="menu">
```

Comment:
We didn't want to make the bet that there wouldn't ever be another `dropdown-menu`. I'm mildly conflicted on adding stuff like this just for tests, but I think it's just about better than that bet.